requirements/constraints.txt

```diff
@@ -8,4 +8,4 @@
 # pin when possible.  Writing an issue against the offending project and
 # linking to it here is good.
 
-Django < 2.3
+Django < 3.3
```

Comment:
Interesting - this change will (unlike the rest of the requirements updates here) actually affect consumers of this module, since `constraints.txt` is pulled into `install_requires`.

That's probably a good thing - this is just more permissive, eh?